Tests/FunctionalTests/fss-search-query.spec.ts

```diff
@@ -27,22 +27,26 @@ describe('Test Search Query Scenario On Search Page', () => {
     expect(await page.innerHTML(pageObjectsConfig.searchPageContainerHeaderSelector)).toEqual(pageObjectsConfig.searchPageContainerHeaderText);
   })
 
-  afterEach(async () => {
+  afterAll(async () => {
```

Comment:
With the proposed `beforeAll`/`beforeEach` changes, this code should be in an `afterEach`
```
await page.close()
await context.close()
```
This code can stay in the `afterAll`
```
await browser.close()
```